app/src/main/java/com/wafflestudio/snutt2/views/logged_in/home/timetable/TimetablePage.kt

```diff
@@ -113,6 +141,55 @@ fun TimetablePage() {
     }
 }
 
+@Composable
+fun VacancyBanner(
+    onClick: () -> Unit,
+    onClose: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(36.dp)
+            .background(SNUTTColors.BannerBlue)
+            .clicks { onClick() }
+            .padding(horizontal = 14.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        RingingAlarmIcon(
+            modifier = Modifier.size(22.dp)
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Row(
+            verticalAlignment = Alignment.Top
+        ) {
+            Text(
+                text = stringResource(R.string.vacancy_banner_text),
+                style = SNUTTTypography.body2.copy(
+                    color = SNUTTColors.AllWhite
+                )
```

Comment:
이거 내가 이렇게 해 놓은게 많은데, 사실 이렇게 하면 됨
```suggestion
                style = SNUTTTypography.body2,
                color = SNUTTColors.AllWhite,
```